core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/UserStoreConfigConstants.java

```diff
@@ -40,10 +40,6 @@
     public static final String claimOperationsSupported = "ClaimOperationsSupported";
     public static final String claimOperationsSupportedDescription = "Whether the userstore supports claim read and write";
     public static final String getClaimOperationsSupportedDisplayName = "Claim Operations Supported";
-    public static final String enableMaxUserLimitForSCIM = "EnableMaxUserLimitForSCIM";
```

Comment:
Since this is not a major release let's avoid removing the constant. Instead, we can deprecate this.